opwen_email_server/api/client_write.py

```diff
@@ -4,14 +4,11 @@
 from opwen_email_server import config
```

Comment:
Why did this (and the files above) change mode from read/write (644) to executable (755)? If this is not intentional, please change back via `chmod 644 /path/to/file`.

Reply:
I have no idea why this happened. I will definitely change this back